tests/test_dte_parse.py

```diff
@@ -4,6 +4,7 @@
 from datetime import date, datetime
 
 import cl_sii.dte.constants
+from cl_sii import __version__
```

Comment:
It is confusing to override implicitly the local magic constants. An example of a [popular pattern](https://www.python.org/dev/peps/pep-0396/#id26) for this:
```
import cl_sii

LIB_CL_SII_VERSION = cl_sii.__version__
```